src/sauce.config.js

```diff
@@ -23,7 +23,7 @@ for (const file of configFiles) {
 const overrides = {
   use: {
     browserName: process.env.BROWSER_NAME, // override browserName with suite browserName
-    headless: process.env.SAUCE_VM ? false : true,
+    headless: process.env.HEADLESS === 'true',
```

Comment:
Don't we still need to distinguish the default behaviour for docker/sauce modes? We don't want to run headed browsers in docker mode, I assume?

Reply:
I cannot recall why we wanted to run headless test in docker mode. And I'm not quite sure if this default setting helps. What about just setting it by config?

Reply:
I think we want headless in docker for efficiency. Assets are generated even in headless mode (I think its why we setup xvfb in the docker image?) so I don't think there's any reason to run headed in docker.

Reply:
set headless for docker as default